include/envoy/access_log/access_log.h

```diff
@@ -77,16 +77,12 @@ class Instance {
   virtual ~Instance() = default;
 
   /**
-   * Log a completed request.
-   * @param request_headers supplies the incoming request headers after filtering.
-   * @param response_headers supplies response headers.
-   * @param response_trailers supplies response trailers.
-   * @param stream_info supplies additional information about the request not
-   * contained in the request headers.
+   * Log a request depending on the AccessLog::Filter configuration.
    */
-  virtual void log(const Http::HeaderMap* request_headers, const Http::HeaderMap* response_headers,
-                   const Http::HeaderMap* response_trailers,
-                   const StreamInfo::StreamInfo& stream_info) PURE;
+  virtual void maybeLog(const Http::HeaderMap* request_headers,
+                        const Http::HeaderMap* response_headers,
+                        const Http::HeaderMap* response_trailers,
+                        const StreamInfo::StreamInfo& stream_info) PURE;
```

Comment:
I prefer leaving this as `log` to avoid churn in the API.

Reply:
Sg, but do you have any better ideas for the other function? maybeLog/log was clear to me, not sure what log/___ should be.

Reply:
log/emitLog perhaps?